lib/screenplay/places/builder.ex

```diff
@@ -0,0 +1,518 @@
+defmodule Screenplay.Places.Builder do
+  @moduledoc """
+  Module used to build the Screenplay config from
+  Showtime and PA/ESS screen configurations. Final configuration is
+  saved to a Nebulex cache.
+
+  Config is rebuilt periodically.
+  """
+
+  alias Screenplay.Places
+  alias Screenplay.Places.Place
+  alias Screenplay.Places.Place.{PaEssScreen, ShowtimeScreen}
+  alias Screenplay.ScreensConfig, as: ScreensConfigStore
+  alias ScreensConfig.{Screen, Solari}
+  alias ScreensConfig.V2.Departures.{Query, Section}
+
+  alias ScreensConfig.V2.{
+    BusEink,
+    BusShelter,
+    Busway,
+    Departures,
+    Dup,
+    Footer,
+    GlEink,
+    Header,
+    PreFare
+  }
+
+  use GenServer
+
+  @sl_route_ids ~w[741 742 743 746 749 751]
+
+  @polling_interval :timer.minutes(15)
```

Comment:
suggestion: 5 minutes? Much more responsive but hopefully still easy on resources/API limits.

This also makes me think perhaps we should check the modification timestamps of the config files this pulls, keep a record (in the state of this module maybe) of when they were last changed, and only go through the whole process if there's been a change. That could be a future improvement though.

Reply:
Yeah 5 minutes would feel less frustrating. 

That's a good idea for an improvement. Could probably go a step further and implement API response caching in Screenplay so we can catch changes to stops and routes. Obviously those change much less frequently, but would hate for us to miss a change just cause our screen data has been changed.